src/common/countries/index.ts

```diff
@@ -0,0 +1,21 @@
+import countries from "./countries";
+import countriesEnglishJSON from "./english";
+
+export interface CountryInterface {
+    countryCode: string;
+    callingCode: string;
+    mask: string;
+}
+
+export interface CountriesLanguageInterface {
+    [index: string]: string;
+}
```

Comment:
_Technically_ you can enforce that all countries are included with TypeScript
```suggestion
export type CountriesLanguageInterface = {
    [index in keyof typeof countriesEnglishJSON]: string;
}
```